modules/ecs/cluster/variables.tf

```diff
@@ -32,4 +32,15 @@ variable "tags" {
   default = {
     terraform = true
   }
+}
+
+variable "app_name" {
+  description = "Name of the app"
+  type        = string
+}
+
+
+variable "kms_key_arn" {
```

Comment:
I don't think either of these variables (`app_name ` and `kms_key_arn`) are used anymore so we dont need to add them.